Resolved Bug 2822 : Design System > Code Snippet > Language Dropdown is not as per Figma Design.

rameshwarkamandar opened the pull request:
## Description
> Resolve the issues on Code Snippet > Language Dropdown is not as per Figma Design.
> Make the changes to /rds-comp-code-snippet.scss, /rds-comp-code-snippet.tsx, /rds-button-dropdown.scss, /rds-button-dropdown.tsx file.
 
## Screenshots:
<img width="1462" height="973" alt="image" src="https://github.com/user-attachments/assets/eeae9b84-f0d6-47fc-8d64-f27c2ac98a1b" />
<img width="1471" height="982" alt="image" src="https://github.com/user-attachments/assets/db9b0e1b-f9d8-4b57-9da7-6350d0ab6538" />

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

rameshwarkamandar committed:
Enhance code snippet and dropdown components UI

Improved the RdsCompCodeSnippet component by adding a language selection dropdown using RdsButtonDropdown, and updated styles for better alignment and accessibility. Added new SCSS for RdsButtonDropdown and refactored its logic to support custom button styles and non-radio options.

supriyartambewaiin committed:
Merge branch 'new-solution' into users/rameshwar/2822-code-snippet-language-dropdown-is-not-as-per-figma-design

supriyartambewaiin committed:
Merge branch 'new-solution' into users/rameshwar/2822-code-snippet-language-dropdown-is-not-as-per-figma-design

supriyartambewaiin commented:
When any language selected from dropdown code format also should be change as per selected language.

rameshwarkamandar committed:
Add support for sample code snippets by language

Enhanced the RdsCompCodeSnippet component to accept a sampleCodeSnippets prop, allowing users to switch between different code samples based on selected language. Updated the Storybook story to provide multiple language code samples for demonstration.

rameshwarkamandar committed:
Merge branch 'development' into users/rameshwar/2822-code-snippet-language-dropdown-is-not-as-per-figma-design

rameshwarkamandar committed:
Merge branch 'development' into users/rameshwar/2822-code-snippet-language-dropdown-is-not-as-per-figma-design

supriyartambewaiin committed:
Merge branch 'development' into users/rameshwar/2822-code-snippet-language-dropdown-is-not-as-per-figma-design
